docs/source/data-access-api/calibration-files.rst

```diff
@@ -0,0 +1,146 @@
+and reprocessing should usually use the same file as the first processing.
+
+These two types of files are mostly used for creating the filename and processing conventions, so if your calibration doesn't neatly fit into
+either category, that's okay. The main thing is that all files should try and follow the conventions in this document.
+
+Specific examples
+^^^^^^^^^^^^^^^^^
+
+As an example, we will be looking at the Mag calibration files. These are supplemental files provided by the instrument team for use in
+L1 and L2 processing.
+
+The first file (L1B rotation matrices) is expected to change rarely,
+with the most recent version being used for all data. This file contains rotation calibration matrices from
+the boom to the instrument. These files would only change if there was a physical change to the boom or if there is
+some more accurate information available, so they are universal calibration files.
+
+The second file contains matrices which are applied to create the L2 and L2pre data. They are updated
+approximately monthly, and contain a start and end time for validity. During reprocessing, the data file with the most
+recent version and a time range that includes the data will be used. Since these are files which specifically cover a time range,
+they are time window calibration files.
+
+These two cases cover the two main types of calibration files. They both will be uploaded to the SDC
+with the API, and the correct file to use will be determined by the filename at processing time.
+
+File name format
+^^^^^^^^^^^^^^^^
+
+The file name for each file should be the following format::
+
+    <instrument>_<calibration_name>_<start_date>_<end_date>_<version>.<extension>
+
+Since these names are only used in instrument processing, they are flexible and can be adapted to
+accommodate different needs. This is more of a rule of thumb than a strict requirement.
+
+* ``<instrument>``: The instrument name associated with the calibration file
+* ``<calibration_name>``: A descriptive name for the calibration file which distinguishes between
+  other calibration files used by the instrument. Rather than "mag-calibration", this should be similar to
+  "mag-rotation-matrices" or "mag-l2pre-calibration." Multiple words should be split by hyphens.
+* ``<start_date>``: The start time of the validity of the calibration file, in the format "YYYYMMDD". This is required for
+  time window calibration files, and is optional for universal calibration files, or files that should be applied in all time
+  ranges where there isn't a file with a higher version available.
+* ``<end_date>``: The end time of the validity of the calibration file, in the format "YYYYMMDD". This is optional for files,
+  with the understanding that if end_date is not provided, the file is valid until a file with a later start_date and no end_date.
```

Comment:
Would we ever expect multiple calibration files per-day of the same type? In other words, do we need to consider adding a `HHMMSS` timestamp here as well for finer granularity?

Reply:
🤷  I wasn't sure. I figure if anyone will care it will be MAG and I can ask them once we have finished SDC in team discussions on it and get that figured out quickly.